Source/swiftlint/Helpers/CompilerArgumentsExtractor.swift

```diff
@@ -95,15 +95,15 @@ extension Array where Element == String {
                 "-serialize-diagnostics",
                 "-emit-dependencies",
                 "-use-frontend-parseable-output"
-            ].contains($0)
+            ].contains($0) // swiftlint:disable:this shorthand_argument
```

Comment:
Defining a named argument is less to type than adding the command to disable the rule. 😉 
```suggestion
            ].contains(arg)
```